README.md

```diff
@@ -1,33 +1,79 @@
-  * Lambda.
-  * CloudWatch (for logging).
-  * IAM (for creating roles).
-  * Elasticsearch.
-* Pelion account with access to API key.
-* Command terminal with access to Curl.
+- AWS account with access to:
+  - API Gateway.
+  - Lambda.
+  - CloudWatch (for logging).
+  - IAM (for creating roles).
+  - Elasticsearch.
+- Pelion account with access to an Application Access key.
+- Command terminal with access to Curl.
+- A device running Pelion Device Management Client that is connected and registered to Pelion (For example, a device running the Pelion example application at [mbed-os-example-pelion](https://github.com/armmbed/mbed-os-example-pelion))
 
 ## Setup
 
+Before setting up the notification channel, first configure the AWS components.  
+
+- First, login to the [AWS Management Console](http://console.aws.amazon.com).  
+
 ### AWS CloudWatch Access
 
-The Lambda creation tool can fail to create a REST API trigger in API Gateway due to missing permissions. Follow this guide to enable CloudWatch logging from API Gateway: https://aws.amazon.com/premiumsupport/knowledge-center/api-gateway-cloudwatch-logs/.
+- Follow [this guide](https://aws.amazon.com/premiumsupport/knowledge-center/api-gateway-cloudwatch-logs/) to enable CloudWatch logging from API Gateway. This will enable logs that can be useful when debugging issues.  
 
 ### AWS Elasticsearch
 
-On the Elasticsearch dashboard, create a new domain with resource settings suitable for the project. For testing purposes, the smallest cluster setting should be sufficient and covered by the free tier. Use "Allow open access to domain" for testing. Note that anyone with the autogenerated URL (including the Lambda function) will have permission to read and write to this instance. 
+- In the AWS console, search for **Elasticsearch** and open the Elasticsearch service dashboard. 
+
+- Click **Create a new domain**. 
 
-Take a note of the URL labled "Endpoint", since this will be used by the Lambda function for data insertion, and the URL labled "Kibana", which will be used for visualizing data. 
+Choose these settings:  
 
-Open Kibana and go to the "Dev Tools" section. In the console, create three new indices using the commands below one at a time:
+- Deployment type
+  - Deployment type: Development and testing
+  - Elasticsearch Version: 7.9
+- Configure Domain
+  - Elasticsearch domain name: *Any name you choose (e.g. pelion-data)* 
+  - Instance type: t2.small.elasticsearch
+  - Number of nodes: 1
+  - Data nodes storage type: EBS
+  - EBS volume type: General Purpose (SSD)
+  - EBS volume size: 10 GiB
+- Configure access and security
+  - Public access: Enabled 
+  - Domain access policy: Allow open access to the domain
+  - Require HTTPS: Enabled
+
+Notes:  
+- Features not specifically mentioned can be disabled. 
+
+- For testing purposes, the smallest cluster setting should be sufficient. Note that this still may not be covered by the AWS free tier, but the cost should be minimal. 
```

Comment:
Perhaps also mention that the ES version number is not fixed to 7.9, but only that is has been tested with 7.9.

Reply:
fixed in 4b755d7